lib/cisco.py

```diff
@@ -827,7 +838,8 @@ def _BuildTokens(self):
                          'owner'}
 
     supported_sub_tokens.update({'option': {'established',
-                                            'tcp-established'},
+                                            'tcp-established',
+                                            'fragments'},
```

Comment:
I think what you are looking for is 'is-fragment' not 'fragments'
https://github.com/google/capirca/blob/24568835dd916652824cf55f25cc66a3a16e716a/lib/aclgenerator.py#L361

Reply:
This is one of the things that makes Cisco so "lovable"... "is-fragment" is a keyword to be used with "fragment-type", but when you want to make an ACE exclusively specific for fragments, you just need to add "fragments" keyword to it, and then it will be evaluate only when fragment offset is not 0. This PR is to support this second "fragments".